File: 241125/2의 거듭제곱수/pow-of-2.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.

    int n;
    int prod = 1;
    int x=0;
    scanf("%d", &n);

    while(1){
        if (prod==n){
            break;
        }
        prod *=2;
        x++;
    }
    printf("%d", x);
    return 0;
}

File: 241128/2의 거듭제곱수/pow-of-2.py
# x가 몇번으로 늘어나야 N과 값이 같아지는지 찾으면 됨

n = int(input())
x=0
prod =1

while True :
    if (prod == n) :
        break
     
    prod = prod * 2
    x+=1

print(x)